services/fhir.ts

```diff
@@ -226,7 +222,7 @@ export async function saveFHIRResources<R extends AidboxResource>(
                     request: {
                         method: resource.id ? 'PUT' : 'POST',
                         url: `/${resource.resourceType}${resource.id ? '/' + resource.id : ''}`,
-                        ...(resource.id && versionId ? { ifMatch: versionId } : {}),
+                        params: resource.id && versionId ? versionId : undefined,
```

Comment:
Leave it as it was before `...(resource.id && versionId ? { ifMatch: versionId } : {}),`